vulkaninfo/vulkaninfo.h

```diff
@@ -108,14 +108,14 @@ struct VulkanException : std::runtime_error {
 };
 #define THROW_VK_ERR(func_name, err) throw VulkanException(func_name, __FILE__, __LINE__, err);
 
-// global configuration
```

Comment:
Was this an accidental removal? 
Also, it would be nice if it was documented *why* these bools are here instead of in main(). Specifically, that the window's WAIT_FOR_CONSOLE_DESTROY macro needs to know if the program is printing to a console (eg human readible output).

Reply:
All good points, and cleaned up.